pkg/tagger/collectors/kubernetes_metadata_mapper.go

```diff
@@ -55,7 +55,12 @@ func (c *KubeMetadataCollector) getTagInfos(pods []*kubelet.Pod) []*TagInfo {
 		} else {
 			metadataNames, err = c.dcaClient.GetKubernetesMetadataNames(po.Spec.NodeName, po.Metadata.Namespace, po.Metadata.Name)
 			if err != nil {
-				log.Tracef("Could not pull the metadata map of po %s on node %s from the Datadog Cluster Agent: %s", po.Metadata.Name, po.Spec.NodeName, err.Error())
+				log.Debugf("Could not pull the metadata map of po %s on node %s from the Datadog Cluster Agent: %s", po.Metadata.Name, po.Spec.NodeName, err.Error())
+				continue
+			}
+			if len(metadataNames) == 0 {
+				log.Tracef("No cluster metadata for pod %s on node %s from the Datadog Cluster Agent", po.Metadata.Name, po.Spec.NodeName)
```

Comment:
do we want this trace log when we're not using the DCA as well? If so we could move this block before the range

Reply:
adapting the message as well